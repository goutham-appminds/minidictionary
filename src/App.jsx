import React from 'react';
import Search from './screens/search';

function App() {
  return <Search />;
}

export default App;
