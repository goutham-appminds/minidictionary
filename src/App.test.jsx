/* eslint-disable no-undef */
import React from 'react';
import {
  render,
} from '@testing-library/react';
import App from './App';

describe('screens/search', () => {
  test('Should handle serach with empty input', () => {
    expect(render(<App />)).toMatchSnapshot();
  });
});
