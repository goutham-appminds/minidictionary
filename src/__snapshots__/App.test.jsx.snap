// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screens/search Should handle serach with empty input 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="page"
      >
        <h1>
          MINI DICTIONARY
        </h1>
        <input
          class="text"
          placeholder="search here.."
        />
        <button
          class="btn"
          data-testid="search"
          type="button"
        >
          Search
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="page"
    >
      <h1>
        MINI DICTIONARY
      </h1>
      <input
        class="text"
        placeholder="search here.."
      />
      <button
        class="btn"
        data-testid="search"
        type="button"
      >
        Search
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
